tests/test_SklearnMLPConverter.py

```diff
@@ -0,0 +1,57 @@
+"""
+Tests scikit-learn's MLPClassifier and MLPRegressor converters.
+"""
+
+import numpy as np
+import unittest
+from sklearn.datasets import load_diabetes, load_iris
+from sklearn.model_selection import train_test_split
+from sklearn.neural_network import MLPClassifier, MLPRegressor
+
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import FloatTensorType
+from test_utils import dump_data_and_model
+
+
+class TestSklearnCalibratedClassifierCVConverters(unittest.TestCase):
+
+    def test_model_mlp_classifier_binary(self):
```

Comment:
you should have test for every supported activation function

{‘identity’, ‘logistic’, ‘tanh’, ‘relu’}, default ‘relu’

Reply:
Makes sense, I'll add them.

Reply:
Added.